packages/arb-token-bridge-ui/src/components/TransactionsTable/TransactionsTableDepositRow.tsx

```diff
@@ -76,17 +76,21 @@ function DepositRowTime({ tx }: { tx: MergedTransaction }) {
     )
   }
 
-  return <span>{tx.resolvedAt || tx.createdAt || 'N/A'}</span>
+  return (
+    <span className="whitespace-nowrap">
+      {tx.resolvedAt || tx.createdAt || 'N/A'}
```

Comment:
not introduced by this PR but the `Time` here is confusing because some is the time of the L1 tx, some is the time of the L2 tx so they don't seem to be sorted by "most recent" correctly when in fact they are in the correct order
<img width="662" alt="image" src="https://user-images.githubusercontent.com/13184582/216055585-ea368625-6f46-44ea-82d1-343de29204c2.png">